tests/baselines/clip_roberta.json

```diff
@@ -45,13 +45,13 @@
                         "--remove_unused_columns False",
                         "--warmup_steps 0",
                         "--weight_decay 0.1",
-                        "--save_strategy epoch",
-                        "--use_hpu_graphs_for_training",
-                        "--use_hpu_graphs_for_inference",
-                        "--dataloader_num_workers 16",
-                        "--distribution_strategy fast_ddp",
+                        "--save_strategy no",
+                        "--use_hpu_graphs",
+                        "--dataloader_num_workers 2",
```

Comment:
can you update this to 16?

Reply:
kibana/Readme all use num_worker to 2, and 16 doesn't make any difference on throughput. Let's keep it with 2.